client/app/deskaas/deskaas-details/deskaas-details.html

```diff
@@ -139,18 +139,17 @@ <h4 class="oui-header_4 cui-tile__title" data-translate="common_tile_shortcut"><
                 <div class="cui-tile h-100">
                     <h4 class="oui-header_4 cui-tile__title" data-translate="vdi_task_current"></h4>
                     <div class="cui-tile__body cui-tile__body_custom cui-tile__body_custom-spaced">
-                        <oui-table data-rows="DeskaasDetailsCtrl.tasksHandler.cleanTasks">
-                            <column title="'vdi_task_id' | translate" property="taskId" sortable="asc"></column>
-                            <column title="'vdi_task_name' | translate" property="displayName" sortable></column>
-                            <column title="'vdi_task_state' | translate" property="displayState" sortable></column>
-                            <column title="'vdi_task_modification_date' | translate" property="lastModificationDate" sortable>
+                        <oui-datagrid data-rows="DeskaasDetailsCtrl.tasksHandler.cleanTasks">
```

Comment:
Question : when should we use data prefix and when we shouldn't ?

Reply:
done, everywhere!